### PR TITLE
Confirm password field in signup page #336

### PR DESCRIPTION
**Pull request by shraddha761**
solves #336

### CODE REVIEW

**Comment by shraddha761** [comment]
@narayan954 please have a look I made the proposed changes

**Comment by narayan954** [comment]
> @narayan954 please have a look I made the proposed changes

why have you created a separate branch and new pr, you should've just updated that pr :/

File: src/pages/Signup.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const SignupScreen = () => {
   const [username, setUsername] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [Confirmpassword, setConfirmPassword] = useState("");
```
Comment (narayan954) [inline src/pages/Signup.jsx:25]: use camelCase please, instead of Confirmpassword use confirmPassword. It's a convention

**Comment by shraddha761** [comment]
I am beginner to open source I try to update same branch with creating another PR but fail to do it

File: src/pages/Signup.jsx [hunk 2]
```diff
@@ -36,6 +37,13 @@ const SignupScreen = () => {
     setShowPassword(!showPassword);
   };
 
+  const [showConfirmPassword, setConfirmShowPassword] = useState(false);
```
Comment (narayan954) [inline src/pages/Signup.jsx:40]: change setConfirmShowPassword -> setShowConfirmPassword

**Review by narayan954 — CHANGES_REQUESTED** [review]
small change

**Comment by narayan954** [comment]
> I am beginner to open source I try to update same branch with creating another PR but fail to do it

That's where you should ask, if you don't try, how'd you learn? I commented on that pull request to the problem you'll be facing. You needed to force push the branch. It's similar to the command you use to push, you just have to add -f to the command. For example `git push -f` instead of `git push`

**Comment by shraddha761** [comment]
@narayan954 I made the changes please have a look

**Comment by narayan954** [comment]
> @narayan954 I made the changes please have a look

Sure, on it!

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**